### PR TITLE
Remove feedback and corrections links

### PR DESCRIPTION
**Pull request by edcous**
This commit removed the i18n translations for:

- correction
- correction_link
- report_missing
- report_missing

as well as removing the references to those in the webpack and html code.

Link to Deploy Preview: https://deploy-preview-350--vaccinatethestates.netlify.app/

---

### Manual Testing (QA)

_(Note: Use your best judgement for time management. If this PR is a minor content adjustment, you might not invest in the full list of QA checks below. But for medium-large PRs, we recommend a thorough review.)_

- [ ] I solemnly swear that I QA'd my change. My change does not break the site on `localhost` nor staging.

#### Home Page
- [ ] Geolocation works
- [ ] Searching by zip works
- [ ] Cards show useful information
  - [ ] Cards address links to Google Maps
  - [ ] Cards can be expanded
- [ ] Zooming out gets us back to blank slate text

#### Embed
- [ ] Query parameters for zip `?zip={zipcode}&zoom={zoom}`
- [ ] Query parameters for lat and lng works `?lat={lat}&lng={lng}&zoom={zoom}`
- [ ] Searching with search box in map works

#### About Us
- [ ] Organizers show up and randomize on page load

### CODE REVIEW

**Review by obra — DISMISSED** [review]

**Comment by edcous** [comment]
On the deploy preview on my phone instead of a symbol in the card expand it just says &rarr;.

831050C3-6740-4148-BCE7-678CF367C050

File: webpack/templates/siteCard.handlebars [hunk 1]
```diff
@@ -151,9 +151,11 @@
                 {{#if lastVerified}}
                     <div class="text-xs">{{t "verified_at" lastVerified=lastVerified}}</div>
                 {{/if}}
-                {{t "correction" }}
-                <a href="https://airtable.com/shrdt7TsVi76kTzYt?prefill_Location+ID={{id}}" target="_blank">{{t "correction_link"}}</a>
             </div>
         </div>
+        <div class="flex flex-row flex-wrap justify-between items-end px-4 py-2 text-sm">
+            <p>{{t "card_out_of_date"}}</p>
+            <p><a href="https://vaccines.gov">{{t "shutdown_forward_no_link"}}</a></p>
```
Comment (skalnik) [inline webpack/templates/siteCard.handlebars:158]:
```suggestion
            <p><a href="https://vaccines.gov">{{{t "shutdown_forward_no_link"}}}</a></p>
```

Should let the `&rarr;` turn into `→` properly

I'd also suggest putting this outside of the `<details></details>` element, so it shows up even if the user doesn't expand the information.

Comment (edcous) [reply webpack/templates/siteCard.handlebars:158]: Should the details just be removed entirely then?

Comment (edcous) [reply webpack/templates/siteCard.handlebars]: resolved by #af5f6d5

**Review by eliblock — APPROVED** [review]
Thank you for wrapping that up @edcous!

**Pull request merged**